packages/runtime/container-runtime/src/summary/README.md

```diff
@@ -1,3 +1,7 @@
+---
+title: Summarization
+---
```

Comment:
Once you follow the patterns I noted, this "front matter" can live only in the docs file.

Reply:
done.